templates/store/snap-details.html

```diff
@@ -197,17 +197,17 @@ <h4>Share this snap</h4>
       </div>
 
       <div class="p-strip is-shallow">
-        <div class="row {% if normalized_os %}u-equal-height{% endif %}">
+        <div class="row {% if normalized_os %}u-equal-height{% endif %}" id="js-chart-row">
```

Comment:
Where possible for JS hooks we should try and use the `data-js` attribute rather than id's and classes.
Then in the code you can do `document.querySelector("[data-js='chart-row']")`

Reply:
totally agree with that. Actually I forgot to remove this id, because is not used anywhere. Just used it to test something and then forgot to remove it.